DomClasses/circle.js

```diff
@@ -0,0 +1,17 @@
+import { createCircle } from "./domInteractions.js"
+import { randomIntFromInterval } from "../Helpers/mathHelper.js"
+
+export class Circle {
+    constructor(widthBounceDiv,heightBounceDiv,leftBounceDiv,topBounceDiv){
+        this.circleRadius = randomIntFromInterval(40,100)
+        this.dom = createCircle(this.circleRadius)
+        this.x = leftBounceDiv+randomIntFromInterval(0,widthBounceDiv-parseInt(this.circleRadius))
+        this.y = topBounceDiv+randomIntFromInterval(0,heightBounceDiv-parseInt(this.circleRadius))
+        this.dom.style.left=this.x+"px"
+        this.dom.style.top=this.y+"px"
+        this.step = randomIntFromInterval(1,10),
+        this.direction = randomIntFromInterval(0,1),
+        this.slope = Math.random()
```

Comment:
The constructor does too many things. It should only initialize the properties of this class, like the circleRadius, x, y, step, direction and slope. The others are DOM interactions, and they should be made through separate in the domInteractions file, like `addCircleToDOM` or `updateCircilePositionInDOM`.